src/containers/MapContainer/index.tsx

```diff
@@ -116,8 +117,7 @@ const MapContainer = () => {
       />
 
       <MapLegend onToggleCategory={handleToggleCategory} />
-      {selectedLayer === 'devices' && <DeviceDetails device={selectedItem} onDeviceDetailsClose={clearSelection} />}
-      {selectedLayer === 'cameras' && <CameraAreaDetails device={selectedItem} onDeviceDetailsClose={clearSelection} />}
+      {selectedLayer && <DeviceDetails device={selectedItem} isAreaCamera={selectedLayer === 'cameras'} onDeviceDetailsClose={clearSelection} />}
```

Comment:
Can we move the isAreaCamera inside the component? Has no relevance here. I would just pass the selectedLayer name to the component.

Reply:
we only need the details to be rendered when  selectedLayer has a value: "cameras" or "devices". otherwise the details are always visible

Reply:
How about
`{selectedLayer && <DeviceDetails device={selectedItem} selectedLayer={selectedLayer} onDeviceDetailsClose={clearSelection} />}`?

Then you can do the rendering inside the the DeviceDetails depending on the type

Reply:
ja dat kan ik doe het wel